packages/relay/src/lib/eth.ts

```diff
@@ -669,12 +669,9 @@ export class EthImpl implements Eth {
                   currentTimestamp = mirrorAccount.balance.timestamp;
                 }
 
-                let transactionsInTimeWindow = await this.mirrorNodeClient.getTransactionsForAccount(
-                  mirrorAccount.account,
-                  block.timestamp.to,
-                  currentTimestamp,
-                  requestId
-                );
+                const transactionsInTimeWindow: any = mirrorAccount.transactions.filter((tx: any) => {
```

Comment:
Nice idea.
Challenge here like it was before is that this is a best guess. 
If the number of transactions is greater than the page limit we may be missing credits and debits and therefore missing paging logic.

In this PR i think you should update `this.mirrorNodeClient.getAccount` or add a `this.mirrorNodeClient.getAccountWithPageLimit()` that makes the call and sets the limit=100 (the page limit)
If not you'll only get 25

Reply:
Can add a ticket for the paging aspect to address later

Reply:
Updated.

Reply:
Agreed.  Basically the pagination handles this issue.  My latest update checks for the need to fetch the next page and handles it if needed.  The latest mirrornode API returns 100 transactions per call on the account endpoint, and a brief look at mainnet suggests that the 100 transactions within the 15 minutes window, causing the need to load the next page, would be more of an exceptional case rather than the norm, so I would say this is an optimization.